loader/vk_loader_platform.h

```diff
@@ -28,14 +28,22 @@
 #include <winsock2.h>
 #endif  // _WIN32
 
+#if defined(__FreeBSD__)
+#ifndef __BSD_VISIBLE
+#define __BSD_VISIBLE 1
```

Comment:
Defining `__BSD_VISIBLE` is a gross hack that nulifies standard compliance. Better disable `_XOPEN_SOURCE` or replace `alloca` et al. with POSIX equivalent(s).

Reply:
Nevermind, I __BSD_VISIBLE forgot this was discussed before.